docker/docker-compose.yml

```diff
@@ -32,8 +61,22 @@ services:
       retries: 5
     volumes:
       - postgres:/var/lib/postgresql/data
+
   redis:
-    image: redis:alpine
+    image: docker.io/library/redis:alpine
+    # cap_drop:
```

Comment:
Why with comments? Either add it or leave it out 🙂

Reply:
Removed in https://github.com/ghostfolio/ghostfolio/pull/3614/commits/75a0cb301e886645838ebac32c9ebafa4c58b6a1